docs/pages/guides/using-nextjs.md

```diff
@@ -147,12 +144,12 @@ This is Vercel's preferred method for deploying Next.js projects to production.
 
 ### Polyfill setImmediate
 
-> 💡 Fixes `setImmediate is not defined` error.
+> Fixes `setImmediate is not defined` error.
 
 A lot of libraries in the React ecosystem use the `setImmediate()` API (like `react-native-reanimated`), which Next.js doesn't polyfill by default. To fix this you can polyfill it yourself.
 
 - Install: `yarn add setimmediate`
-- Import in **pages/_app.js**, at the top of the file:
+- Import in **pages/\_app.js**, at the top of the file:
```

Comment:
Hmm, was this change intentional?

Reply:
Nope. It sneaked past me. I guess my VSCode config are broken after recently updating it. I'll revert this. Thanks for pointing this out 👍